src/run

```diff
@@ -19,8 +19,8 @@ base_dir=$(cd "`dirname "$loc"`" && pwd)
 # Determine os version and choose correspondent .sh or .bat file accordingly.
 os_version="`uname -a`"
 
-if [[ $os_version == *"MINGW"* ]]; then
-  exec "$base_dir/target/start.bat" "$@"
-else
+if [[ $os_version == *"MINGW"* ]]; then  
+  cmd //C "$base_dir/target/start.bat" "$@"
```

Comment:
If you want to see an diff between my changes + your changes <---> master then take a look at https://github.com/wpeter-sfdc/djinni/pull/1